app/routes/wow.weekly-price-group-delta.tsx

```
@@ -0,0 +1,1256 @@
+}
+`
+
+// Overwrite default links in the root.tsx
+export const links: LinksFunction = () => [
+  {
+    rel: 'canonical',
+    href: 'https://saddlebagexchange.com/wow/weekly-price-group-delta'
+  },
+  {
+    rel: 'stylesheet',
+    href: 'data:text/css,' + encodeURIComponent(styles)
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)